images_config/ks-robot-master.cfg

```diff
@@ -0,0 +1,270 @@
+apt-get install ros-indigo-roscpp -y
+apt-get install ros-indigo-rosbash -y
+apt-get install ros-indigo-ros -y
+apt-get install ros-indigo-care-o-bot-robot -y
+%end
+
+################ SETUP DEFAULT BASH ENVIROMENT ################ 
+%post --interpreter=/bin/bash
+wget -O /etc/cob.bash.bashrc https://raw.githubusercontent.com/ipa320/setup_cob4/master/cob-pcs/cob.bash.bashrc.b
+%end
+
+################ INSTALL SHUTDOWN ################ 
+%post --interpreter=/bin/bash
+wget -O /usr/sbin/cob-shutdown https://raw.githubusercontent.com/ipa320/setup_cob4/master/scripts/cob-shutdown
+chmod +x /usr/sbin/cob-shutdown
+sed -i 's/etc\/acpi\/powerbtn.sh/usr\/sbin\/cob-shutdown/g' /etc/acpi/events/powerbtn
+echo "%users ALL=NOPASSWD:/usr/sbin/cob-shutdown" >> /etc/sudoers
+%end
+
+################ NETWORK SETUP ################ 
+%post --interpreter=/bin/bash --logfile /root/ks-post.log
+apt-get -y remove biosdevname -y --force-yes
+update-initramfs -u
+update-grub
+
+(
+  wget -O /etc/network/interfaces https://raw.githubusercontent.com/ipa320/setup_cob4/master/cob-pcs/networkInterfacesB1
+) 2>&1 | /usr/bin/tee /root/post-install.log
+chvt 1
+touch /etc/network/interfaces.orig
+cp /etc/network/interfaces /etc/network/interfaces.orig
+echo "mv /etc/network/interfaces.orig /etc/network/interfaces && ifup -a && sed -i '/fixnet.sh/d' /etc/rc.local && rm -f /fixnet.sh" > /fixnet.sh
+sed -i '/exit 0/ibash /fixnet.sh' /etc/rc.local
+wget -O /etc/network/interfaces https://raw.githubusercontent.com/ipa320/setup_cob4/master/cob-pcs/networkInterfacesB1
+/etc/init.d/networking restart
+%end
+
+################ INSTALL CANDUMP TOOLS ################ 
+%post
+initctl stop candump2mongodb
+rm /etc/init/candump2mongodb.conf
+wget -O /usr/local/bin/socket_buffer.py https://raw.githubusercontent.com/ipa320/setup_cob4/master/scripts/socket_buffer.py
+chmod +x /usr/local/bin/socket_buffer.py
+wget -O /etc/init/candump_any.conf https://raw.githubusercontent.com/ipa320/setup_cob4/master/scripts/candump_any.conf
+initctl start candump_any
+%end
+
+################ INSTALL NOMACHINE ################ 
+%post
+apt-get install pulseaudio -y
+wget -O /root/nomachine_5.3.9_6_amd64.deb http://download.nomachine.com/download/5.3/Linux/nomachine_5.3.9_6_amd64.deb
+dpkg -i /root/nomachine_5.3.9_6_amd64.deb
+%end
+
+################ INSTALL NETDATA TOOLS ################ 
+%post --interpreter /bin/bash
+apt-get install zlib1g-dev uuid-dev libmnl-dev gcc make git autoconf autoconf-archive autogen automake pkg-config curl -y
+git clone https://github.com/firehol/netdata.git --depth=1 /root/netdata
+cd /root/netdata
+yes | ./netdata-installer.sh
+%end
+
+################ ROBOT WORK SPACE ################ 
+%post
+apt-get install python-catkin-tools -y
+su robot -c "mkdir -p /u/robot/git/care-o-bot/src"
+source /opt/ros/indigo/setup.bash
+rosdep init
+su robot -c "rosdep update"
+su robot -c "cd /u/robot/git/care-o-bot/ && catkin init"
+su robot -c "cd /u/robot/git/care-o-bot/ && catkin build"
```

Comment:
we have used this command not in cfg files but manually , robot workspace module of master cfg file  is not executed while booting due to creating of /u/robot dir after first session of login. I'm still working on same